### DISCUSSION TITLE
How to attach InteractionBehaviour script to instantiated ECS entity?

### DISCUSSION THREAD

**Discussion by sunillnaik** [leapmotion/UnityModules#1187] [Q&A] [answered]
This may be silly question, but I have a sample which instantiates sphere and want to attach the InteractionBehaviour script to the ECS entities. Is it possible like we do with GameObject?

**Comment by Ultraleap-Dan** [comment, marked as answer]
Currently this is not something that we have looked at. We will put this on our backlog to be investigated and will update you when we have any new information available.
